modules/reference/pages/rpk/rpk-connect/rpk-connect-studio-pull.adoc

```diff
@@ -0,0 +1,33 @@
+= rpk connect studio pull
+
+Run deployments configured within a Redpanda Connect session.
+
+When a Studio session has one or more deployments added this command will synchronize with the session and obtain a deployment assignment. The assigned deployment will then determine which configs from the session to download and execute.
+
+When either changes are made to files of an assigned deployment, or when a new deployment is assigned, this service will automatically download the new config files and execute them, replacing the previous stream running.
+
+== Usage
+
+[,bash]
+----
+rpk connect studio pull [command options] [arguments...]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--session, -s |- | The session ID to synchronize with.
+
+|--name |- | An explicit name to adopt in this instance, used to identify its connection to the session. Each running node must have a unique name, if left unset a name is generated each time the command is run.
+
+|--token, -h      |- | A token for the session, used to authenticate requests. If left blank the environment variable `BSTDIO_NODE_TOKEN` will be used instead.
+
+|--token-secret     |- | A token secret the session, used to authenticate requests. If left blank the environment variable `BSTDIO_NODE_SECRET` will be used instead.
+
+|--send-traces      |- | Whether to send trace data back to Studio during execution. This is opt-in and is used as a way to add trace events to the graph editor for testing and debugging configs. This is a very useful feature but should be used with caution as it exports information about messages passing through the stream (default: false). 
```

Comment:
```suggestion
|--send-traces      |- | Whether to send trace data back to Studio during execution. This is an opt-in way to add trace events to the graph editor for testing and debugging configs. This is a very useful feature, but use it with caution, because it exports information about messages passing through the stream (default: false). 
```